packages/ods-react/src/style/_drawer.scss

```diff
@@ -0,0 +1,137 @@
+@use './overlay';
```

Comment:
add this `_drawer` file to the forward list in the `index.scss` file in this same dir